bittensor/commands/register.py

```diff
@@ -127,7 +128,7 @@ def add_args(parser: argparse.ArgumentParser):
         bittensor.subtensor.add_args(register_parser)
 
     @staticmethod
-    def check_config(config: "bittensor.config"):
+    async def check_config(config: "bittensor.config"):
```

Comment:
If we change `check_config` to a coroutine here, we're going to need to update `cli.check_config`.